Things.md

```diff
@@ -169,4 +170,12 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing Classes
+
+A Thing Class is the definition of a single type of Thing (i.e. earlier we used CastleFireball). Each unique in-game Thing has a Thing Class defining all of its properties. There can, of course, be multiple instances of a single Thing Class.
```

Comment:
> i.e. earlier we used CastleFireball

Not a huge fan of this phrasing. Try "such as CastleFireball" from earlier.

Also, CastleFireball isn't a good thing to keep using because it's Mario-specific. Prefer Block or Tree or Bush or etc.